pcsx2/PAD/Host/StateManagement.cpp

```diff
@@ -241,7 +241,21 @@ u8 pad_poll(u8 value)
 					b1=b1 & 0x1f;
 #endif
 
-				uint16_t buttons = g_key_status.GetButtons(query.port);
+				uint32_t buttons = g_key_status.GetButtons(query.port);
+				if (!test_bit(buttons, 16) && !pad->modeLock)
```

Comment:
```suggestion
				if (!test_bit(buttons, PAD_ANALOG) && !pad->modeLock)
```

Avoids the magic number if the indices ever change.